runtime/common/xwalk_switches.h

```diff
@@ -29,6 +29,8 @@ extern const char kPpapiFlashPath[];
 extern const char kPpapiFlashVersion[];
 #endif
 
+extern const char kUserDataDir[];
```

Comment:
From http://peter.sh/experiments/chromium-command-line-switches/, we can see it's a general switch.